test/user_test.dart

```diff
@@ -116,4 +116,299 @@ Future<void> main([List<String>? args]) async {
     final user = await app.logIn(Credentials.emailPassword(testUsername, testPassword));
     expect(user.profile.email, testUsername);
   });
+
+  baasTest('User.apiKeys.create creates and reveals value', (configuration) async {
+    final app = App(configuration);
+    final user = await getIntegrationUser(app);
+    final apiKey = await user.apiKeys.create('my-api-key');
+
+    expect(apiKey.isEnabled, true);
+    expect(apiKey.name, 'my-api-key');
+    expect(apiKey.value, isNotNull);
+    expect(apiKey.id, isNot(ObjectId.fromValues(0, 0, 0)));
+  });
+
+  baasTest('User.apiKeys.create with invalid name returns error', (configuration) async {
+    final app = App(configuration);
+    final user = await getIntegrationUser(app);
+    await expectLater(
+        () => user.apiKeys.create('Spaces are not allowed'),
+        throwsA(isA<AppException>()
+            .having((e) => e.message, 'message', contains('can only contain ASCII letters, numbers, underscores, and hyphens'))
+            .having((e) => e.linkToServerLogs, 'linkToServerLogs', contains('logs?co_id='))
+            .having((e) => e.statusCode, 'statusCode', 400)));
+  });
+
+  baasTest('User.apiKeys.fetch with non existent returns null', (configuration) async {
+    final app = App(configuration);
+    final user = await getIntegrationUser(app);
+
+    final key = await user.apiKeys.fetch(ObjectId());
+    expect(key, isNull);
+  });
+
+  void expectApiKey(ApiKey? fetched, ApiKey expected) {
```

Comment:
```suggestion
  void _expectApiKey(ApiKey? fetched, ApiKey expected) {
```

Reply:
This is a local function, not a class-level one, so prefixing it with an underscore is a lint violation.

Reply:
oh ok. missed that one.